dnf5daemon-server/threads_manager.hpp

```diff
@@ -67,9 +67,11 @@ class ThreadsManager {
                 } catch (const sdbus::Error & ex) {
                     reply = call.createErrorReply(ex);
                 } catch (const std::exception & ex) {
-                    reply = call.createErrorReply({dnfdaemon::ERROR, ex.what()});
+                    sdbus::Error err{dnfdaemon::ERROR, ex.what()};
+                    reply = call.createErrorReply(std::move(err));
```

Comment:
Since the new constructor is explicit, it must be called explicitly. No more is needed.
Replace the line `reply = call.createErrorReply({dnfdaemon::ERROR, ex.what()});` with `reply = call.createErrorReply(sdbus::Error(dnfdaemon::ERROR, ex.what()));`

It also applies to the next occurrence.

Reply:
Thanks! Will change it.